qwiic_template.py

```diff
@@ -170,506 +91,37 @@ def __init__(self, address=None, i2c_driver=None):
         else:
             self._i2c = i2c_driver
 
-
-        # create a dictionary to stash our calibration data for the sensor
-        self.calibration={}
-
-        self.t_fine=0
-
-        self._referencePressure = 101325.0
-
-    # ----------------------------------
-    # is_connected()
-    #
-    # Is an actual board connected to our system?
+        # TODO: Initialize any variables used by this driver
 
     def is_connected(self):
         """
-            Determine if a BME280 device is conntected to the system..
-            :return: True if the device is connected, otherwise False.
-            :rtype: bool
+        Determines if this device is connected
+
+        :return: `True` if connected, otherwise `False`
+        :rtype: bool
         """
+        # Check if connected by seeing if an ACK is received
+        # TODO: If the device has a product ID register, that should be
+        # checked in addition to the ACK
         return qwiic_i2c.isDeviceConnected(self.address)
 
     connected = property(is_connected)
```

Comment:
@gigapod Thoughts on just removing this? Kinda goofy to have `connected` and `is_connected()` that do the same thing IMO